src/dashboards/components/VEO.scss

```diff
@@ -25,6 +25,8 @@
   align-items: stretch;
   width: 100%;
   flex: 1 0 0;
+  padding-left: $cf-space-l;
+  padding-right: $cf-space-l;
```

Comment:
<img width="1536" alt="Screen Shot 2022-09-26 at 4 49 53 PM" src="https://user-images.githubusercontent.com/18511823/192400300-2dc6ed6e-699d-4919-a5b7-2f56623049d7.png">

It had weird padding issue.